x-pack/legacy/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/apm/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,267 @@
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import React, { Fragment } from 'react';
+import {
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiCallOut,
+  EuiText
+} from '@elastic/eui';
+import { Monospace } from '../components/monospace';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { statusTitle } from './common_apm_instructions';
+import { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } from 'ui/documentation_links';
+
+export function getApmInstructionsForEnablingMetricbeat(product, _meta, {
+  esMonitoringUrl,
+  hasCheckedStatus,
+  checkingMigrationStatus,
+  checkForMigrationStatus,
+  autoCheckIntervalInMs
+}) {
+  const securitySetup = (
+    <Fragment>
+      <EuiSpacer size="m"/>
+      <EuiCallOut
+        color="warning"
+        iconType="help"
+        title={(
+          <EuiText>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.apmInstructions.metricbeatSecuritySetup"
+              defaultMessage="If security features are enabled, there may be more setup required.{link}"
+              values={{
+                link: (
+                  <Fragment>
+                    {` `}
+                    <EuiLink
+                      href={`${ELASTIC_WEBSITE_URL}guide/en/apm/reference/${DOC_LINK_VERSION}/configuring-metricbeat.html`}
+                      target="_blank"
+                    >
+                      <FormattedMessage
+                        id="xpack.monitoring.metricbeatMigration.apmInstructions.metricbeatSecuritySetupLinkText"
+                        defaultMessage="View more information."
+                      />
+                    </EuiLink>
+                  </Fragment>
+                )
+              }}
+            />
+          </EuiText>
+        )}
+      />
+    </Fragment>
+  );
+  const installMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.apmInstructions.installMetricbeatTitle', {
+      defaultMessage: 'Install Metricbeat on the same server as the APM server'
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <EuiLink
+            href={`${ELASTIC_WEBSITE_URL}guide/en/apm/metricbeat/${DOC_LINK_VERSION}/metricbeat-installation.html`}
+            target="_blank"
+          >
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.apmInstructions.installMetricbeatLinkText"
+              defaultMessage="Follow the instructions here"
+            />
+          </EuiLink>
+        </p>
+      </EuiText>
+    )
+  };
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.apmInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the beat x-pack module in Metricbeat'
```

Comment:
Maybe capitalize `Beat`?

Reply:
I'm not sure in this case as the module is called `beat-xpack`. I can go either way I guess. Anyone else have thoughts here?

Reply:
IMHO, `Beat` is still a proper name in the same way that we capitalize `Metricbeat`. Though, I will grant you that the naming scheme that was chosen makes this feel a little awkward. :)

Reply:
Changed!